packages/KModal/KModal.vue

```diff
@@ -15,6 +15,11 @@
           <div class="modal-header">
```

Comment:
```suggestion
          <div class="modal-header" :class="helpText ? 'mb-3' : 'mb-4'">
```

Our use case has a `12px` bottom margin here, so I'd suggest adding this conditional bottom margin and removing the styling in the style tags:
```
margin-bottom: var(--spacing-md, spacing(md));
```

Reply:
@dustinryerson Having trouble visualizing what you mean here. May be better to chat on zoom.

Reply:
Fixed